lib/net.js

```diff
@@ -1311,6 +1311,8 @@ function lookupAndConnect(self, options) {
   const host = options.host || 'localhost';
   let { port, autoSelectFamilyAttemptTimeout, autoSelectFamily } = options;
 
+  validateString(host, 'host');
```

Comment:
Should the name here be `options.host` instead of `host`, as in the other validation cases here?

Reply:
Yes, that makes sense for consistency. Thank you!